libfwupdplugin/fu-plugin.c

```diff
@@ -1159,9 +1159,11 @@ gboolean
 fu_plugin_runner_update_prepare (FuPlugin *self, FwupdInstallFlags flags, FuDevice *device,
```

Comment:
maybe these runner functions should rename too

Reply:
Good idea, fixed.